src/main/resources/application.yml

```
@@ -30,4 +52,4 @@ spring:
   config:
     import: classpath:application-prod.properties
     activate:
-      on-profile: prod
+      on-profile: prod
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)